### PR TITLE
Add a build_status value for each notebook image

### PR DESCRIPTION
**Pull request by mroman-redhat**

### CODE REVIEW

File: jupyterhub_singleuser_profiles/images.py [hunk 1]
```diff
@@ -47,6 +48,14 @@ def get_default(self):
                 return image.name
 
         return image_list[0].name if len(image_list) else None
+    
+    def get_image_build_status(self, imagestream_name):
+        build_list = self.openshift.get_builds()
+        for build in build_list.items:
+            #if imagestream_name in build.metadata.name:
+            if imagestream_name == build.spec.output.to.name:
```
Comment (mroman-redhat) [inline jupyterhub_singleuser_profiles/images.py:56]: Is this the optimal way to gather the build belonging to an imagestream?

**Comment by mroman-redhat** [comment]
This PR also required the Jupyterhub service account to have permission to `get` builds

File: jupyterhub_singleuser_profiles/images.py (hunk 1 above)
Comment (lucferbux) [reply jupyterhub_singleuser_profiles/images.py:56]: maybe @vpavlin could give you more insight

Comment (vpavlin) [reply jupyterhub_singleuser_profiles/images.py:56]: Yeah, I think that's fine

**Comment by vpavlin** [comment]
LGTM

**Pull request merged**